### PR TITLE
Fix all little inconsistencies in the json command file descriptions

### LINKED ISSUE #40: Fix all the little inconsistencies in the json command file descriptions
Ideally all of the description should be:
1. Full sentences with capitalization
2. Include punctuation.

### PR DESCRIPTION
**Pull request by pragnesh**
~Partial~ fix for #40.
~I want to get feedback before making more changes,~ Please review.

### CODE REVIEW

File: src/commands/role.json [hunk 1]
```diff
@@ -28,27 +28,27 @@
...
+                            "description": "Connected replicas.",
                             "type": "array",
                             "items": {
                                 "type": "array",
                                 "minItems": 3,
                                 "maxItems": 3,
                                 "items": [
                                     {
-                                        "description": "replica ip",
+                                        "description": "Replica ip.",
```
Comment (zuiderkwast) [inline src/commands/role.json]:
```suggestion
                                        "description": "Replica IP.",
```

File: src/commands/role.json [hunk 2]
```diff
@@ -65,48 +65,48 @@
                             "const": "slave"
                         },
                         {
-                            "description": "ip of master",
+                            "description": "Ip of master.",
```
Comment (zuiderkwast) [inline src/commands/role.json]:
```suggestion
                            "description": "IP of master.",
```

**Review by zuiderkwast — APPROVED** [review]
Very good. You may need to run make and check in some generated file.

**Comment by pragnesh** [comment]
> Very good. You may need to run make and check in some generated file.

I tried building documentation but getting error
```
#utils/generate-commands-json.py
Traceback (most recent call last):
  File "/data2/valkey/valkey/utils/generate-commands-json.py", line 117, in <module>
    p = subprocess.Popen([args.cli, '-h', args.host, '-p', str(args.port), '--json', 'command'], stdout=subprocess.PIPE)
  File "/home/pragnesh/.pyenv/versions/3.10.8/lib/python3.10/subprocess.py", line 971, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/home/pragnesh/.pyenv/versions/3.10.8/lib/python3.10/subprocess.py", line 1847, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: '/data2/valkey/valkey/src/redis-cli'
```
I am not sure what is correct way to build documentation. Please suggest.

**Comment by zuiderkwast** [comment]
The binary `redis-cli` has been renamed? There are many PRs now.

**Pull request merged**